molecule/default/tests/test_create_snapshot_on_instance.py

```diff
@@ -32,23 +33,19 @@ def test_create_snapshot_of_an_instance(host):
         "network_name": private_net,
     }
 
-    utils.create_instance(data_image, host)
-    assert (utils.verify_asset_in_list('server', instance_name, host))
+    helpers.create_instance(data_image, host)
+    sleep(30)  # TODO: will find out a better way to avoid implicit sleep.
+    assert (helpers.asset_is_in_the_list('server', instance_name, host))
 
     # Verify the new instance is ACTIVE.
-    assert (utils.get_expected_value('server', instance_name, 'status', 'ACTIVE', host))
-
-    # Shutdown the newly created instance
-    utils.stop_server_instance(instance_name, host)
-
-    # Verify that the instance is shutdown
-    assert (utils.get_expected_value('server', instance_name, 'status', 'SHUTOFF', host))
+    assert (helpers.get_expected_value('server', instance_name, 'status', 'ACTIVE', host, 20))
+    sleep(60)  # TODO: will find out a better way to avoid implicit sleep.
```

Comment:
I don't understand what this sleep call accomplishes. The server is verified to be 'ACTIVE' in the previous line. Is there another condition that needs to be satisfied before the snapshot can be created?